src/gps/GPS.cpp

```diff
@@ -650,21 +672,6 @@ int32_t GPS::runOnce()
     return isAwake ? GPS_THREAD_INTERVAL : 5000;
```

Comment:
The value GPS_THREAD_INTERVAL does not match the one in the comment above

Reply:
That's... odd. Been that way for a while, too. We can try setting it to 200.